src/Locks.ts

```diff
@@ -26,18 +26,30 @@ type LockEventMap = {
 };
 
 export default class Locks extends EventEmitter<LockEventMap> {
+  // locks tracks the local state of locks, which is used to determine whether
+  // a lock's status has changed when processing presence updates.
+  //
+  // The top-level map keys are lock identifiers, the second-level map keys are
+  // member connectionIds, and the values are the state of locks those members
+  // have requested.
+  private locks: Map<string, Map<string, Lock>>;
+
   constructor(
     private space: Space,
     private presenceUpdate: (update: PresenceMember['data'], extras?: any) => Promise<void>,
   ) {
     super();
+    this.locks = new Map();
   }
 
   get(id: string): Lock | undefined {
-    for (const member of this.space.members.getAll()) {
-      const request = member.locks.get(id);
-      if (request && request.status === 'locked') {
-        return { request, member };
+    const locks = this.locks.get(id);
+    if (!locks) {
+      return;
+    }
```

Comment:
I've noticed that throughout this library we use single line syntax:
```suggestion
    if (!locks) return;
```
I think it might be worth keeping a consistent style throughout?

Reply:
Ah yes, updated in 028354d, thanks.